config.json

```diff
@@ -2464,6 +2464,32 @@
           "multiple-clause-matching"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "circular-buffer",
+        "name": "Circular Buffer",
+        "uuid": "535d64c9-95f7-4f59-b7b6-d459337b82b0",
+        "prerequisites": [
+          "atoms",
+          "tuples",
+          "lists",
+          "structs",
+          "agent",
+          "processes",
+          "pids",
+          "case",
+          "cond",
+          "if",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "guards",
+          "errors"
+        ],
+        "practices": [
+          "processes",
+          "errors"
```

Comment:
@neenjaw Do you remember how we defined practicing/requiring "errors"? Is it about the ok/error-tuple pattern, or is it about using `raise` (`exceptions` is definitely about defining custom ones)?

Reply:
I think it is implemented to exercise `raise`

Reply:
I'm not opposed to this addition though, it is an accepted way of handling errors